### PR TITLE
add blendui v0.0.4 with npm auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/Clouda-team/BlendUI
**Watch 52
  Star 280
 Fork 105**
@PeterDaveHello  could u help me check this PR for #5853  ? thank you!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR 
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update) https://github.com/Clouda-team/BlendUI/issues/22
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/blendui/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "Javascript",
+    "webview",
+    "web"
+  ],
+  "npmName": "blendui",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(BlendHybridUI-*)"
```
Comment (PeterDaveHello) [inline ajax/libs/blendui/package.json]: Should exclude number, not the prefix ...

Comment (maruilian11) [inline ajax/libs/blendui/package.json]: so should i add each released number... ?

Comment (PeterDaveHello) [inline ajax/libs/blendui/package.json]: Haven't you read the doc? Why not just use RegExp?

https://github.com/isaacs/node-glob#glob-primer 

> `[...]` Matches a range of characters, similar to a RegExp range. If the first character of the range is `!` or `^` then it matches any character not in the range.

File: ajax/libs/blendui/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "Javascript",
+    "webview",
+    "web"
+  ],
+  "npmName": "blendui",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(BlendHybridUI-[0-9].[0-9].[0-9]*)"
```
Comment (maruilian11) [inline ajax/libs/blendui/package.json]: @PeterDaveHello  is that ok?

Comment (PeterDaveHello) [inline ajax/libs/blendui/package.json]: Are you sure about there will only be a single number? Why do you use`+([0-9])` instead of `[0-9]`?

**Comment by maruilian11** [comment]
@PeterDaveHello updated.

**Comment by maruilian11** [comment]
ping @PeterDaveHello

**Pull request merged**